screener-e2e/examples/test/configs/wdio.shared.conf.js

```diff
@@ -0,0 +1,37 @@
+exports.config = {
+    // ====================
+    // Runner Configuration
+    // ====================
+    runner: 'local',
+    // ==================
+    // Specify Test Files
+    // ==================
+    specs: [
+        './test/specs/**/*.js'
+    ],
+    // ============
+    // Capabilities
+    // ============
+    maxInstances: 100,
+    // capabilities can be found in the `wdio.local.chrome.conf.js` or `wdio.sauce.conf.js`
+    // ===================
+    // Test Configurations
+    // ===================
+    logLevel: 'debug',
+    bail: 0,
+    baseUrl: 'https://www.saucedemo.com/',
+    waitforTimeout: 10000,
+    connectionRetryTimeout: 90000,
+    connectionRetryCount: 3,
+    framework: 'jasmine',
```

Comment:
After pull Luis's PR, to pass the sauce test, you also need to comment out this line as well.